packages/page-staking/src/Payouts/index.spec.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright 2017-2021 @polkadot/app-staking authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import {PayoutsPage} from '../../test/pages/payoutsPage';
+import {StakerState} from "@polkadot/react-hooks/types";
+
+describe('Payouts', () => {
+  let payoutsPage: PayoutsPage;
+  const someOwnedValidator: StakerState = {
+    controllerId: 'someId',
+    hexSessionIdNext: '0x1112',
+    hexSessionIdQueue: 'sth',
+    isLoading: false,
+    isOwnController: false,
+    isOwnStash: false,
+    isStashNominating: false,
+    isStashValidating: false,
+    sessionIds: [],
+    stashId: 'someStashId'
+  }
+
+  beforeEach(() => {
+    payoutsPage = new PayoutsPage();
+  });
+
+  describe('empty view', () => {
+    it('renders `My validators/stashes` toggle', async () => {
+      payoutsPage.renderPage();
+
+      await payoutsPage.expectText('My validators');
+      await payoutsPage.expectText('My stashes');
+    });
+
+    it('when no own validators, then `My validators` is disabled and `My stashes` selected', async () => {
```

Comment:
```suggestion
    it('when no own validators, then `My validators` is disabled and not selected, async () => {
```
We don't actually test whether "My stashes" is selected. So let's either test it, or omit from the the test name.

Reply:
we do actually (or I don't understand sth here)
See: https://github.com/polkadot-js/apps/pull/5522/files#diff-f03687e4905ff7fd437cadfd8c39ac3291e0d64e25789ae96841860dd586ed52R111-R112

Reply:
The check you referenced is done on "My validators", not on "My stashes".

Reply:
ah, now I understand. sure :)